Makefile

```
@@ -147,6 +158,10 @@ $(APP).elf: check_tamago
 
 
 $(APP)_manifest:
+	@if [ "${APPLET_PRIVATE_KEY}" == "" ] || [ ! -f "${APPLET_PRIVATE_KEY}" ]; then \
+		echo 'You need to set the APPLET_PRIVATE_KEY variable to a valid note verifier key path'; \
```

```suggestion
		echo 'You need to set the APPLET_PRIVATE_KEY variable to a valid note signing key path'; \
```

I know you're just testing how closely I'm reviewing it :-)

---

Exactly, well done - good job :)